clang/test/SemaSYCL/intel-fpga-global-const.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsyntax-only -ast-dump -verify -pedantic %s | FileCheck %s
```

Comment:
during merge conflict, i saw this lit test failure. I have updated it here.

Reply:
>>basically, we no longer should pass -fsycl to any %clang_cc1 invocation.

Yes, agree

Reply:
I created #3339 before I saw this comment.
In general I think it's better to separate logically independent patches: test fix and attribute refactoring.
I suggest we merge #3339 first and #3326 after that.

Reply:
>>I suggest we merge #3339 first and #3326 after that.

Agree. Thanks @bader for fixing the test.

Reply:
> I suggest we merge #3339 first and #3326 after that.

Sounds sensible!